File: game/src/main/scala/falcun/net/api/fonts/FalcunFont.scala
package falcun.net.api.fonts

trait FalcunFont {
  def drawString(text: String, x: Number, y: Number, color: Int, underline: Boolean): Unit

  def getStringWidth(text: String): Number

  def stringHeight(text: String): Number

  def stringHeight = 9
}

File: client/projects/Forge/src/main/java/falcun/net/api/fonts/FalcunFont.java
package falcun.net.api.fonts;

import java.util.ArrayList;
import java.util.Collections;
import java.util.List;

public interface FalcunFont {

	void drawString(String text, Number x, Number y, int color, boolean underline);

	Number getStringWidth(String text);

	Number stringHeight(String text);

	default Number stringHeight() {
		return 9;
	}

	List<String> getLinesWrapped(String text, int maxWid);

	int size();

}
